bool[4] enable